src/shared/bg_pmove.cpp

```diff
@@ -1289,12 +1285,12 @@ static bool PM_CheckWallRun()
 
 	pm->ps->groundEntityNum = ENTITYNUM_NONE;
 
-	normal = Vec3::Load( trace.plane.normal );
-	velocity = Vec3::Load( pm->ps->velocity );
-
-	velocity += Math::Normalize( normal + Vec3( 0, 0, 0.9 ) ) * jumpMag;
+	vec3_t tmp;
+	VectorSet( tmp, 0.f, 0.f, 0.9f );
+	VectorAdd( tmp, trace.plane.normal, tmp );
+	VectorNormalize( tmp );
+	VectorMA( pm->ps->velocity, jumpMag, tmp, pm->ps->velocity );
```

Comment:
I would write
```suggestion
	vec3_t tmp;
	VectorCopy( trace.plane.normal, tmp );
	tmp[2] += 0.9f;
	VectorNormalize( tmp );
	VectorMA( pm->ps->velocity, jumpMag, tmp, pm->ps->velocity );
```

Reply:
If we're nitpicking initialization, better is `vec3_t tmp = {0, 0, 9}`

Reply:
Considering the reason of this PR, I prefer slipher's suggestion (no need for later changes). Also, still considering the reasons for this, I would prefer to avoid nitpicks, since the idea is to replace vec3_t *soon*.